content/en/docs/spin-operator/tutorials/deploy-on-azure-kubernetes-service.md

```diff
@@ -42,6 +42,7 @@ az group create --name rg-spin-operator \
 # Create an AKS cluster
 az aks create --name aks-spin-operator \
     --resource-group rg-spin-operator \
+    --location germanywestcentral \
```

Comment:
Although resources deployed into a Azure Resource Group inherit the location of the group, I thought it is more self-explaining to explicitly specify the desired location.